README.md

```diff
@@ -18,29 +18,29 @@ Sentinel is an autonomous agent for persisting, processing and automating Dash g
 
 ## Install
 
-These instructions cover installing Sentinel on Ubuntu 16.04 / 18.04.
+These instructions cover installing Sentinel on Ubuntu 18.04 / 20.04.
 
 ### Dependencies
 
-Make sure Python version 2.7.x or above is installed:
+Update system package list and install dependencies:
 
-    python --version
+    $ sudo apt-get update
+    $ sudo apt-get -y install git python3 virtualenv
 
-Update system packages and ensure virtualenv is installed:
+Make sure Python version 3.6.x or above is installed:
 
-    $ sudo apt-get update
-    $ sudo apt-get -y install python-virtualenv
+    python3 --version
 
-Make sure the local DashCore daemon running is at least version 12.1 (120100)
+Make sure the local DashCore daemon running is at least version 15.0 (150000)
 
-    $ dash-cli getinfo | grep version
+    $ dash-cli getnetworkinfo | jq -r .version
```

Comment:
Could instead just ask `dashd` directly via `dashd --version` (or `dashd --version | head -n1` if you want version only, without copyright info)